backend/app/core/conveyor/notebook_builder.py

```diff
@@ -13,6 +13,7 @@ def __init__(self):
         self.conveyor_api = "https://app.conveyordata.com/api/v2"
 
         self.oidc = OIDCConfiguration(
+            oidc_disabled=False,
```

Comment:
Should we use `get_boolean_variable("OIDC_DISABLED", True)` and not in `OIDCConfiguration.__init__`?